examples/aws_efa/nccl_efa.yaml

```diff
@@ -50,6 +66,8 @@ config:
         containers:
         - resources:
             limits:
-              vpc.amazonaws.com/efa: 2
+              vpc.amazonaws.com/efa: 4
+              nvidia.com/gpu: 8
```

Comment:
Do we need this? It seems to be the label/annotation that is set automatically by k8s / SkyPilot.

Reply:
This is to request the gpu resource in Pod spec.

Reply:
Yes, you're right. I will remove them.